STEERING-COMMITTEE.md

```diff
@@ -7,8 +7,7 @@ aliases:
   - /contributing/steering-committee/
 ---
 
-The Knative Steering Committee (KSC) is the ultimate authority for the Knative
-project, and governs all aspects of the project.
+The Knative Steering Committee (KSC) governs all aspects of the project.
 
 The governance of Knative is an open, living document, and will continue to
```

Comment:
Would you also please delete this line? https://github.com/knative/community/pull/187/files#diff-885f133673bb843be731122e9098f958R41 this is handled by elections in our community now.